socorro/collector/throttler.py

```diff
@@ -32,20 +33,11 @@ class LegacyThrottler(RequiredConfig):
               and d.get("ProcessType", "browser") == "browser"''', None),
         # 100% of crashes with comments
         ("Comments", '''lambda x: x''', 100),
-        # 100% of all aurora, beta, esr channels
-        ("ReleaseChannel", '''lambda x: x in ("aurora", "beta", "esr")''', 100),
-        # 100% of all crashes that report as being nightly
-        ("ReleaseChannel", '''lambda x: x.startswith('nightly')''', 100),
-        # 10% of Firefox
-        ("ProductName", 'Firefox', 10),
-        # 100% of Fennec
-        ("ProductName", 'Fennec', 100),
-        # 100% of all alpha, beta or special
-        ("Version", r'''re.compile(r'\..*?[a-zA-Z]+')''', 100),
-        # 100% of Thunderbird & SeaMonkey
-        ("ProductName", '''lambda x: x[0] in "TSC"''', 100),
-        # reject everything else
-        (None, True, 0)
+        # 10% of Firefox release
+        ("*", '''lambda d: d.get("ProductName") == "Firefox" and
+            d.get("ReleaseChannel") == "release"''', 10),
+        # accept everything else
+        (None, True, 100)
```

Comment:
What do you propose we do?

I think the original reasoning behind this was the prospect of using Socorro for a new Mozilla product that was being planned/prototyped and they asked about using Breakpad to dump the crashes with us. It was that discussion to caused us to take a second look at the list of rules and they we spun into a discussion about refactoring/simplifying. 

We **could** hard code the names of projects we expect and if a new project wants to start sending us stuff (using breakpad presumably) we change the code, and have it in production within 48 hours. 
Or we could do it in config. But without a good config management system and having to rely on ssh and consulate I strongly oppose.
Or we could do it in a database which we manage with a admin web app. 

By the way, should be chat about DOS attacks in a private channel? I have some more thoughts on the matter.